packages/date-picker/src/vaadin-month-calendar.js

```diff
@@ -260,6 +288,14 @@ class MonthCalendar extends ThemableMixin(PolymerElement) {
     return weekDayNames;
   }
 
+  __focusedDateChanged(focusedDate, days) {
+    if (days.some((date) => dateEquals(date, focusedDate))) {
+      this.removeAttribute('aria-hidden');
+    } else {
+      this.setAttribute('aria-hidden', 'true');
+    }
```

Comment:
nit: untested (ignore, in case the tests are to be added with the following refactors)

Reply:
Will cover this in the next PR as well.